app/templates/_service_summary_features_and_benefits.html

```diff
@@ -1,4 +1,4 @@
-<p class="service-summary-lede">{{ service.serviceSummary or service.serviceDescription }}</p> {# G-Cloud 9 uses serviceDescription, not serviceSummary. #}
```

Comment:
I think we need to keep this - this template is also used for previous framework iteration service pages, which still exist now.